mod-invoice-storage/schemas/voucher.json

```diff
@@ -0,0 +1,117 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "voucher",
+  "type": "object",
+  "properties": {
+    "id": {
+      "description": "UUID of this voucher",
+      "type": "string",
+      "pattern": "^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[1-5][0-9a-fA-F]{3}-[89abAB][0-9a-fA-F]{3}-[0-9a-fA-F]{12}$"
+    },
+    "systemCurrency": {
+      "description": "The currency in which the voucher was paid",
+      "type": "string"
+    },
+    "invoiceCurrency": {
+      "description": "The currency in which invoice was originally defined",
+      "type": "string"
+    },
+    "exchangeRate": {
+      "description": "Exchange rate used to generate transaction",
+      "type": "number"
+    },
+    "exportToAccounting": {
+      "description": "Export this payment record to a financial system",
+      "type": "boolean"
+    },
+    "voucherType":{
+      "description": "Indicates type of payment to account",
+      "type": "string",
+      "enum": [
+        "Payment",
+        "Pre-Payment",
+        "Credit",
+        "Voucher"
+      ]
+    },
+    "voucherStatus":{
+      "description": "status of the voucher",
```

Comment:
`s` in `status` should be uppercase

Reply:
There is not much description yet, will be added when we have more information